test/src/main/java/org/apache/accumulo/test/functional/GarbageCollectorIT.java

```diff
@@ -186,7 +187,8 @@ public void dontGCRootLog() throws Exception {
   }
 
   private Mutation createDelMutation(String path, String cf, String cq, String val) {
-    Text row = new Text(MetadataSchema.DeletesSection.getRowPrefix() + path);
+    Text row =
+        new Text(MetadataSchema.DeletesSection.getRowPrefix() + SortSkew.getCode(path) + path);
```

Comment:
the pattern `MetadataSchema.DeletesSection.getRowPrefix() + SortSkew.getCode(path) + path` could be it's own utility function, since it's used frequently. Perhaps: `MetadataSchema.DeletesSection.encodeDeleteRow(path)` or similar.

Reply:
good idea, didn't really like having to use the SortSkew in the IT test.